deployment/dockerfile.tools

```diff
@@ -43,4 +43,4 @@ USER www-data
 
 FROM scratch as final
 
-COPY --from=tools_intermediate / /
\ No newline at end of file
+COPY --from=tools_intermediate / /
```

Comment:
Would this _not_ just copy the entire filesystem from the `tools_intermediate` into this `scratch` image? I simply don't understand _why_? doesn't it effectively turn `scratch` into `tools_intermediate`....? why not just continue using `tools_intermediate`, and do w/o `scratch` entirely?